NYPLCardCreator/Classes/AlternativeAddressesViewController.swift

```diff
@@ -32,16 +32,31 @@ final class AlternativeAddressesViewController: TableViewController {
   
   override func viewDidLoad() {
     super.viewDidLoad()
+    
+    let headerView = UIView(frame: CGRect(x: 0, y: 0, width: self.view!.bounds.size.width, height: 80.0))
+    let headerLabel = UILabel()
+    headerView.addSubview(headerLabel)
+    
+    headerLabel.autoPinEdgesToSuperviewEdgesWithInsets(UIEdgeInsetsMake(2.0, 22.0, 2.0, 22.0))
```

Comment:
The same advice regarding magic numbers goes here too. Apple provides a concept of view margins that you can take advantage of via PureLayout that should let you do this in a cleaner way.